### ISSUE TITLE
Unclear RTD project URL

### ISSUE THREAD

**Issue by webknjaz** [FFY00/python-pyproject-metadata#88]
I noticed that the README badge and the metadata point to https://pyproject-metadata.rtfd.io that does not actually exist, while the repo frontpage sidebar links https://pep621.rtfd.io that seems to actually get updates from this repo.

Which one is canonical? Which should be fixed?

**Comment by FFY00** [comment]
Ah, the badge and metadata got renamed when we renamed the project, but we didn't actually change the URL.

Since we are in the PyPA now, I will move the docs to the pypa.io URL and update everything.

**Comment by henryiii** [comment]
I'm updating the broken links to pyproject-metadata.pypa.io, assign that will be set up. Do we need @ewdurbin to help?

**Comment by webknjaz** [comment]
Yes, I think only Ee has access to update the DNS records. As of now, it doesn't point to RTD:
```console
$ dig @1.1.1.1 pyproject-metadata.pypa.io

; <<>> DiG 9.16.42 <<>> @1.1.1.1 pyproject-metadata.pypa.io
; (1 server found)
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 40124
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1232
;; QUESTION SECTION:
;pyproject-metadata.pypa.io.	IN	A

;; AUTHORITY SECTION:
pypa.io.		885	IN	SOA	ns-1411.awsdns-48.org. awsdns-hostmaster.amazon.com. 1 7200 900 1209600 86400

;; Query time: 13 msec
;; SERVER: 1.1.1.1#53(1.1.1.1)
;; WHEN: Tue Mar 26 13:38:54 CET 2024
;; MSG SIZE  rcvd: 140
```

**Comment by webknjaz** [comment]
@henryiii @FFY00 could you clearly state which of the two above RTD projects I mentioned is going to remain canonical so that Ee would know where to point the corresponding CNAME DNS record. I reread the comments and couldn't understand the answer to this question.

It seems like `pyproject-metadata.rtfd.io` has been removed from RTD so my understanding is that we need `pyproject-metadata.pypa.io -> pep621.rtfd.io` CNAME. Please correct me, if you're going to create a new RTD project with a new name instead of reusing what's currently integrated.

**Comment by ewdurbin** [comment]
awaiting confirmation on what DNS is needed.

**Comment by webknjaz** [comment]
@FFY00 @henryiii this seems to be blocked on you deciding what subdomain to use. Any decision?

**Comment by henryiii** [comment]
I'm not a maintainer on https://readthedocs.org/projects/pep621/, only @FFY00 is. I think either that should be used, or it should be renamed to pyproject-metadata.rtfd.io and then used.

**Comment by henryiii** [comment]
Without feedback, let's go with pep621 for now. Assuming it's not too hard to change in the future if we rename.

**Comment by FFY00** [comment]
Sorry for the delay, Github notification are a bit hard to keep track of. I have added you.
